packages/code/src/LanguageSwitcher/LanguageSwitcherExample.tsx

```diff
@@ -98,10 +98,14 @@ export function LanguageSwitcherWithDeprecatedPropsExample({
     <Code
       {...rest}
       language={language.displayName}
-      languageOptions={languageOptions}
-      customActionButtons={customActionButtons}
-      onChange={handleChange}
-      chromeTitle="Title"
+      panel={
```

Comment:
You can remove this example since we don't need it anymore. The example above, `LanguageSwitcherWithPanelExample`, already uses the panel prop.